### PR TITLE
Feature-create-animals-table

### PR DESCRIPTION
**Pull request by maximoortelli**
### Feature-create-animals-table:

_The main goal of the project is to set up the fundamental data structure for a veterinary clinic using a relational database. To achieve this, use a PostgreSQL database to build a table that stores crucial information about the animals. This encompasses tasks such as entering example data into the table and executing diverse queries to retrieve information._

### Requirements:

- [x] _Establish a PostgreSQL database named "vet_clinic."_
- [x] _Construct a table named "animals" with predefined columns to house relevant data._
- [x] _Load the "animals" table with sample data, as outlined in the project guidelines._
- [x] _Formulate SQL queries to accomplish various objectives._
- [x] _Document with a screenshot._
- [x] _Arrange all SQL queries systematically within the "queries.sql" file._
- [x] _Adhere to established standards for both Gitflow and documentation practices._

### Screenhots:

 _In order to complete the requirements I add the screenshots._

### CODE REVIEW

File: queries.sql
Comment (aamirkhan2478) [inline queries.sql:1]: - [x] If you check the requirement then you will see there mentioned take a screenshot of your queries which means you will take a screenshot or screenshots of your all queries and add screenshots or screenshots in your pull request but for now you just add one result but the requirement is to all queries result so kindly fix this issues :slightly_smiling_face:  

> _**Project Requirement**_
> - Take a screenshot of the results of your queries.
> - Remember to add the screenshot with the results of your queries to your Pull Request description.

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @maximoortelli, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql
Comment (maximoortelli) [reply queries.sql:1]: Done, thank you @aamirkhan2478 👍😃

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @maximoortelli && @aamirkhan2478  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Made changes as requested by the previous reviewer :+1: 

## Optional suggestions 💭 

[OPTIONAL]
- You were only required to share the screenshots of the queries but it's fine :sweat_smile: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**